finality-aleph/src/sync/service.rs

```diff
@@ -40,6 +41,8 @@ pub struct Service<
     chain_events: CE,
     justifications_from_user: mpsc::UnboundedReceiver<J::Unverified>,
     additional_justifications_from_user: mpsc::UnboundedReceiver<J::Unverified>,
+    _block_requests_from_user:
```

Comment:
Hm, could you perhaps read from this channel instead and ignore the results? That way if you end up implementing the submissions side first you won't get a memory leak at least.

This might require either keeping the other side of the channel alive or fusing this side (unless it's fused by default?).

Reply:
I'll just mark A0-2409 as blocked by A0-2408.